src/OpenTherm.cpp

```diff
@@ -207,7 +231,7 @@ void OpenTherm::process()
 		}
 	}
 	else if (st == OpenThermStatus::DELAY) {
-		if ((newTs - ts) > 100000) {
+		if ((newTs - ts) > 115000) {
```

Comment:
I took this from the documentation:
`The master must communicate at least every 1 sec (+15% tolerance).`
It seems to me that this is applicable to any timings, since there is rarely a real-time clock anywhere. If you are against it, I will return 100ms.

Reply:
This value not a time range, this is delay between commands. No need to wait an additional 15ms to send the next command.

Reply:
I assumed that some boilers could wait 100ms, but 100ms on the ESP and 100ms on the boiler hardware are not the same, since there is no real time clock. Is it better to remove the extra 15ms?